### DISCUSSION TITLE
Wifi Connection

### DISCUSSION THREAD

**Discussion by plaxtika** [mikey0000/Mammotion-HA#59] [General]
When is planned to integrate the WiFi Connection? 

The BT Range is most of the time not enough!

**Comment by mikey0000** [comment]
It's not far off, busy making changes to the underlying library which can already communicate over WiFi, so the answer is soon. All of course based on my time availability.

**Reply by mikey0000** [reply]
Feel free to join our discord to keep up to date on progress.

**Comment by dodavies** [comment]
Super cool to hear the wifi is coming, I cannot use BT as the range is too little so currently fingers crossed here for wifi before I install

**Comment by zipposi** [comment]
I'm excited to connect via WiFi!  :)

**Comment by mikey0000** [comment]
It's coming, just need it to be stable.
